lib/dgram.js

```diff
@@ -344,6 +346,13 @@ Socket.prototype.send = function(buffer,
   if (typeof callback !== 'function')
     callback = undefined;
 
+  if (typeof address === 'function') {
+    callback = address;
+    address = undefined;
+  } else if (typeof address !== 'undefined' && typeof address !== 'string') {
```

Comment:
I don't think it matters in this case since it would just fail the other part of the conditional if it was `null`.